CMakeLists.txt

```diff
@@ -297,6 +297,9 @@ OCV_OPTION(WITH_GTK "Include GTK support" ON
 OCV_OPTION(WITH_GTK_2_X "Use GTK version 2" OFF
   VISIBLE_IF UNIX AND NOT APPLE AND NOT ANDROID
   VERIFY HAVE_GTK AND NOT HAVE_GTK3)
+OCV_OPTION(WITH_FRAMEBUFFER "Include framebuffer support" OFF
```

Comment:
We can set it to "ON" temporarily in order to enable build on CI.

Reply:
the flag value is replaced with "ON"